bindings/python/examples/client/ledger_nano.py

```diff
@@ -1,22 +1,23 @@
-from iota_client import IotaClient, LedgerNanoSecretManager
+from iota_sdk import Client, LedgerNanoSecretManager, SecretManager
 
 # In this example we will get the ledger status and generate an address
 # To use the ledger nano simulator clone https://github.com/iotaledger/ledger-shimmer-app, run `git submodule init && git submodule update --recursive`,
 # then `./build.sh -m nanos|nanox|nanosplus -s` and use `True` in `LedgerNanoSecretManager(True)`.
 
-# Create an IotaClient instance
-client = IotaClient(nodes = ['https://api.testnet.shimmer.network'])
+# Create a Client instance
+client = Client(nodes = ['https://api.testnet.shimmer.network'])
 
 is_simulator = True
 
-secret_manager = LedgerNanoSecretManager(is_simulator)
+ledger_secret_manager = LedgerNanoSecretManager(is_simulator)
+secret_manager = SecretManager(ledger_secret_manager)
 
 # Get the Ledger Nano status.
-ledger_nano_status = client.get_ledger_nano_status(is_simulator)
+ledger_nano_status = secret_manager.get_ledger_nano_status()
 
 print(f'Ledger Nano status: {ledger_nano_status}')
 
 # Generate public address with custom account index and range.
-address = client.generate_addresses(secret_manager, account_index=0, start=0, end=1)
+address = client.generate_addresses(ledger_secret_manager, account_index=0, start=0, end=1)
```

Comment:
Does this not need a secret_manager ?

Reply:
No, this is still the old client method where the secret manager is provided as JSON
I left it because we use the default bech32 hrp from the node there, but not sure if that's reason enough or if we should also remove it